devtools/conda-recipe/meta.yaml

```diff
@@ -28,12 +29,13 @@ requirements:
     - six
     - openmm
     - parmed
-
+    - tqdm
```

Comment:
I don't have a strong feeling either way, but do we really need this dependence? It's helpful when running the tests locally, but the new tests could be re-written without it.